docs/dev/README.md

```diff
@@ -1,35 +1,73 @@
 ---
-title: Introduction to Agent-based Integrations
+title: Build an Integration
 kind: documentation
+description: Learn how to develop and publish an offering on the Integrations page.
 aliases:
-  - /guides/agent_checks/
-  - /agent/agent_checks
-  - /developers/agent_checks/
+- /guides/agent_checks/
+- /agent/agent_checks
+- /developers/agent_checks/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/README.md"
+further_reading:
+- link: "/developers/"
+  tag: "Documentation"
+  text: "Learn how to develop on the Datadog platform"
 ---
 
+## Overview
+
+This page walks through how Technology Partners can build an out-of-the-box integration using the Datadog Agent or Datadog API. 
+
+The [Integrations page][3] includes integrations built by both Datadog and our Technology Partners, available at no cost to Datadog customers. 
+
+The [Datadog Marketplace][4], on the other hand, is a commercial platform for Technology Partners to _sell_ a variety of offerings, including integrations, software licenses, and professional services to Datadog customers.
+
+## Datadog Integrations
+
+### [Agent-based Integration][2]
+Agent-based integrations use the Datadog Agent to submit data via checks written by the partner. The implementation code for these integrations is hosted by Datadog. Agent Integrations are best suited for collecting data from systems or applications that live in a local area network (LAN) or virtual private cloud (VPC). Writing an Agent integration requires you to publish and deploy your solution as a Python wheel (.whl).
+
+### [API-based Integration][1]
+API-based integrations can enrich a customer's Datadog environment by submitting data from external technologies via the Datadog API. Customers authorize API integrations access to their accounts through OAuth, and partners write and host the implementation code that makes up the integration. API integrations work well for partners building a connector between Datadog and another SaaS platform.
```

Comment:
Can we replace the hazy "enrich a customer's Datadog environment" and be more specific about submitting telemetry like metrics, logs, traces that can be correlated, alerted on, visualized, and analyzed? 

Separately, are we avoiding stating the option to also read data out, not just submit?